static/scripts/util.js

```diff
@@ -0,0 +1,61 @@
+// Utility function to send an AJAX POST request
+
+function sendPostRequest(url, data, successCallback, errorCallback) {
+    $.ajax({
+        type: 'POST',
+        url: url,
+        data: data,
+        success: successCallback,
+        error: errorCallback
+    });
+}
+
+// Function to handle the change event of the yearSelect dropdown
+function handleYearSelectChange(params) {
```

Comment:
Well, actually, after looking more carefully, this can probably be handled in a cleaner fashion by :

1. Adding a `data-redirect` attribute to each ` select > option` with the result of each `url_for`.
3. In the `change` listener below, change the window location to the value of `$(this).find(':selected").data("redirect")`